beacon-chain/rpc/proposer_server.go

```diff
@@ -228,11 +228,10 @@ func (ps *ProposerServer) computeStateRoot(ctx context.Context, block *ethpb.Bea
 	if err != nil {
 		return nil, fmt.Errorf("could not get beacon state: %v", err)
 	}
-	s, err := state.ExecuteStateTransition(
+	s, err := state.ExecuteStateTransitionNoVerify(
```

Comment:
can you add a regression test that a proposed block with an empty signature would not fail this ?

Reply:
It's tested here https://github.com/prysmaticlabs/prysm/pull/3048/files#diff-8b31cb04037af6e80706bcc1372a15b1R79

ill make sure the block has an empty sig